CS342MemberViews/Sources/RajPabari/FunFact.swift

```diff
@@ -0,0 +1,49 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+public struct FunFact: View {
```

Comment:
You don't have to make this view public if you only want to use it internally in your target. No access modifier should be fine here.
```suggestion
struct FunFact: View {
```
You will have to then adapt the other access modifier as well.